src/app/CalculateSalary/selectors.js

```diff
@@ -4,3 +4,12 @@ export const selectFormMain = createSelector(
     (state) => state.Calculate.formMain,
     (form) => form,
 );
```

Comment:
Please add in title a message for describing PR

Reply:
done